data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/configuration/SchedulingParameterConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.opensearch.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+import jakarta.validation.constraints.Min;
+import org.opensearch.dataprepper.parser.DataPrepperDurationDeserializer;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+public class SchedulingParameterConfiguration {
+
+    @JsonProperty("rate")
+    @JsonDeserialize(using = DataPrepperDurationDeserializer.class)
+    private Duration rate;
+
+    @Min(1)
+    @JsonProperty("job_count")
+    private int jobCount = 1;
+
+    @JsonSerialize(using = LocalDateTimeSerializer.class)
+    @JsonDeserialize(using = LocalDateTimeDeserializer.class)
+    @JsonProperty("start_time")
+    private LocalDateTime startTime = LocalDateTime.now();
```

Comment:
I believe we discussed making this an `Instant` in the other PR.

https://github.com/opensearch-project/data-prepper/pull/2661#discussion_r1191741997

Reply:
Thank you @dlvenable for review and comments. We have incorporated the required changes.